smithy-model/src/test/java/software/amazon/smithy/model/knowledge/PropertyBindingIndexTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *   http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.model.knowledge;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.model.shapes.MemberShape;
+import software.amazon.smithy.model.shapes.OperationShape;
+import software.amazon.smithy.model.shapes.ShapeId;
+import software.amazon.smithy.model.validation.ValidatedResult;
+import software.amazon.smithy.model.validation.ValidatedResultException;
+
+public class PropertyBindingIndexTest {
+
+    @Test
+    public void testIndex() {
+        ValidatedResult<Model> vrmodel = Model.assembler()
+                    .addImport(OperationIndexTest.class.getResource("member-property-index.smithy"))
+                    .assemble();
+        Model model = vrmodel.getResult().get();
```

Comment:
why not use the `unwrap` method of the assembler?

Reply:
`unwrap` is called later in the test method to assert throws. I still needed to access the Model to build the index and perform checks on `ValidatedResult<Model>`.